docs/management/admin/admin-pg-ov.asciidoc

```diff
@@ -108,4 +112,4 @@ To filter the Endpoints list, use the search bar to enter a query using *{kibana
 [role="screenshot"]
 image::images/filter-endpoints.png[]
 
-NOTE: The timepicker on the right side of the page allows you to set a time interval to automatically refresh the Endpoints list -- for example, if new endpoints were added or deleted.
+NOTE: The time picker on the right side of the page allows you to set a time interval to automatically refresh the Endpoints list — for example, if new endpoints were added or deleted.
```

Comment:
Small nite - I typically refer to the time picker feature as the "date and time picker" in my docs but I've also seen it labeled as the "date picker" (see the "Host Key Performance Indicators (KPIs) charts" section in this topic) in our docs so might be worth it to come up with a single term for this one. :)

Reply:
Good point! I like "date and time picker" since it's more accurate. And yes, a good term to standardize.